src/Utils.cpp

```diff
@@ -60,70 +65,69 @@ void CUtils::GenerateCert(FILE *pOut, const CString& sHost) {
 	unsigned int uSeed = (unsigned int)time(NULL);
 	int serial = (rand_r(&uSeed) % 9999);
 
-	RSA *pRSA = RSA_generate_key(2048, 0x10001, NULL, NULL);
-	if ((pKey = EVP_PKEY_new())) {
-		if (!EVP_PKEY_assign_RSA(pKey, pRSA)) {
-			EVP_PKEY_free(pKey);
-			return;
-		}
+#ifdef HAVE_RSA_GENERATE_KEY_EX
```

Comment:
The function was added in 0.9.8 betas. I'll remove the "else" code path, dropping support here for older OpenSSL.